float32 base_main
float32 arm_hinge
float32 hand_arm